.github/workflows/build-desktop.yml

```diff
@@ -280,12 +188,120 @@ jobs:
         id: build-desktop
         with:
           os: mac
-      - name: build the app
+      - name: Build the app
         run: |
           eval "$(rbenv init - bash)"
           pnpm build:lld --api="http://127.0.0.1:${{ steps.turborepo-cache-server.outputs.port }}" --token="yolo" --team="foo"
-      - name: upload macOS app
+      - name: Upload macOS app
         uses: actions/upload-artifact@v3
         with:
           name: ${{ steps.build-desktop.outputs.version }}-mac.dmg
           path: ${{ github.workspace }}/apps/ledger-live-desktop/dist/${{ steps.build-desktop.outputs.name }}-${{ steps.build-desktop.outputs.version }}-mac.dmg
+
+  report:
+    needs:
+      [
+        build-desktop-app-linux,
+        build-desktop-app-macos,
+        build-desktop-app-windows,
+      ]
+    runs-on: ubuntu-latest
+    if: always() && !cancelled() && github.event_name == 'workflow_dispatch' && github.event.inputs.sha != ''
+    steps:
+      - uses: actions/github-script@v6
+        name: build summary
+        with:
+          script: |
+            const fs = require("fs");
```

Comment:
This JS script should be in its own .js file. It's hard to maintain js code hidden inside a .yml config file plus you can't write unit tests for an inline script. (usually all code deserves unit tests).